apps/aehttp/src/aehttp_dispatch_int.erl

```diff
@@ -169,3 +179,14 @@ sign_and_push_to_mempool(Tx) ->
     ok = aec_tx_pool:push(SignedTx),
     lager:debug("pushed; peek() -> ~p",
                 [pp(aec_tx_pool:peek(10))]).
+
+safe_decode(Type, Enc) ->
```

Comment:
when this is in `aec_base58c` we could save duplicate code from `apps/aehttp/src/aehttp_dispatch_ext.erl`